docs/changehistory/NextVersion.md

```diff
@@ -50,3 +51,29 @@ The Xml and JSON representations of a custom attribute (and related TypeScript i
 ## Improve the performance of the ECSchemaRpcLocater
 
 Improve the performance of the ECSchemaRpcLocater by making all of the underlying ECSchemaRpcInterface methods GET by default so responses are cached by default. Previously each client had to set the methods to be GET or they would default to POST and were not cached.
+
+## Mathematical equations formatting
```

Comment:
This is really parsing mathematical operations and calculating it, not formatting right?

```suggestion
## Mathematical operation parsing
```

Reply:
You're right, it should be operation everywhere, equations are algebra stuff. Thanks !